API.md

```diff
@@ -131,6 +131,7 @@ Subscribes a react component's state directly to a store key
 | [mapping.initWithStoredValues] | <code>Boolean</code> | If set to false, then no data will be prefilled into the  component |
 | [mapping.waitForCollectionCallback] | <code>Boolean</code> | If set to true, it will return the entire collection to the callback as a single object |
 | [mapping.selector] | <code>String</code> \| <code>function</code> | THIS PARAM IS ONLY USED WITH withOnyx(). If included, this will be used to subscribe to a subset of an Onyx key's data.       If the selector is a string, the selector is passed to lodashGet on the sourceData. If the selector is a function, the sourceData and withOnyx state are       passed to the selector and should return the simplified data. Using this setting on `withOnyx` can have very positive performance benefits because the component       will only re-render when the subset of data changes. Otherwise, any change of data on any property would normally cause the component to re-render (and that can       be expensive from a performance standpoint). |
+| [mapping.initialValue] | <code>Any</code> | THIS PARAM IS ONLY USED WITH withOnyx(). If included, this will be passed as the initial hydrated value for the mapping. It will allow the component to render eagerly while data is being fetched from the DB. Note that it will not cause the component to have the `loading` prop set to true. |
```

Comment:
> the initial hydrated value for the mapping

What exactly does "hydrated" mean? 😅 

Is there a better term we can use for this that has the same meaning? I can only think of someone drinking a lot of water.

Reply:
Standard [React lingo](https://react.dev/reference/react-dom/hydrate). But open to suggestions.

Reply:
Heh, yeah, but what does it mean to you? Like, let's say you were explaining this to a 7-year-old.

Reply:
I was talking with another co-worker about this and ChatGPT for the win!

<img width="842" alt="Screenshot_2023-09-15_at_10 44 15_AM-2023-09-15 17_46_10 411" src="https://github.com/Expensify/react-native-onyx/assets/1228807/1ad408e9-cf5d-45fe-87ce-5486a862b4e9">

So, I think it would be helpful to change this to say:

> If included, this will be passed to the component so that something can be rendered while data is being fetched from the DB. Note that it will not cause the component to have the `loading` prop set to true.